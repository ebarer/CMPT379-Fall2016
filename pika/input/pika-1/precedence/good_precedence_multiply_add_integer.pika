exec {

  print 36 * 6 + 28, _n_.
  print 54 * 68 + 35, _n_.
  print 43 * 93 + 96, _n_.
  print 36 * 71 + 51, _n_.
  print 63 * 16 + 82, _n_.
  print 27 * 53 + 50, _n_.
  print 1 * 44 + 92, _n_.
  print 43 * 62 + 25, _n_.
  print 83 * 82 + 29, _n_.
  print 86 * 94 + 95, _n_.
  print 9 * 53 + 67, _n_.
  print 18 * 47 + 34, _n_.

  print 17 + 27 * 78, _n_.
  print 73 + 57 * 45, _n_.
  print 24 + 48 * 76, _n_.
  print 30 + 32 * 59, _n_.
  print 19 + 30 * 30, _n_.
  print 68 + 46 * 92, _n_.
  print 18 + 12 * 2, _n_.
  print 80 + 8 * 21, _n_.
  print 3 + 81 * 60, _n_.
  print 77 + 30 * 89, _n_.
  print 99 + 34 * 73, _n_.
  print 50 + 59 * 67, _n_.

  print 30 * 21 + 54 * 34, _n_.
  print 71 * 26 + 31 * 13, _n_.
  print 86 * 13 + 83 * 61, _n_.
  print 54 * 23 + 14 * 17, _n_.
  print 84 * 8 + 79 * 72, _n_.
  print 87 * 54 + 26 * 83, _n_.
  print 87 * 11 + 35 * 73, _n_.
  print 83 * 10 + 6 * 76, _n_.
  print 7 * 99 + 44 * 25, _n_.
  print 65 * 1 + 70 * 51, _n_.
  print 53 * 79 + 62 * 97, _n_.
  print 57 * 27 + 15 * 52, _n_.

  print 40 + 9 * 43 + 72, _n_.
  print 12 + 92 * 44 + 5, _n_.
  print 53 + 40 * 70 + 69, _n_.
  print 77 + 59 * 3 + 79, _n_.
  print 27 + 65 * 49 + 65, _n_.
  print 95 + 4 * 88 + 20, _n_.
  print 23 + 9 * 12 + 99, _n_.
  print 32 + 94 * 68 + 98, _n_.
  print 77 + 79 * 57 + 28, _n_.
  print 67 + 99 * 65 + 77, _n_.
  print 59 + 22 * 36 + 64, _n_.
  print 48 + 18 * 67 + 68, _n_.

}

