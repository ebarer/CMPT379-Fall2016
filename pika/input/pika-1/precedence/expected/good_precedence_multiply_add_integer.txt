244
3707
4095
2607
1090
1481
136
2691
6835
8179
544
880
2123
2638
3672
1918
919
4300
42
248
4863
2747
2581
4003
2466
2249
6181
1480
6360
6856
3512
1286
1793
3635
10201
2319
499
4065
2922
333
3277
467
230
6522
4608
6579
915
1322
